server/CompositeStorage/lib/index.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Storage,
+  IndexStorage,
+  ComponentsStorage,
+  Component,
+  File,
+  Index,
+  ComponentTree,
+  ComponentGetter,
+  Maybe
+} from '@dynamico/driver';
+
+export class CompositeStorage implements Storage {
```

Comment:
I think it's a composition storage
```suggestion
export class CompositionStorage implements Storage {
```